examples/sveltekit/package.json

```diff
@@ -11,8 +11,8 @@
     "format": "prettier --write ."
   },
   "devDependencies": {
-    "@sveltejs/adapter-auto": "next",
-    "@sveltejs/kit": "1.0.0-next.589",
+    "@sveltejs/adapter-auto": "1.0.0-next.65",
+    "@sveltejs/kit": "1.0.0-next.428",
```

Comment:
Why was this one seemingly downgraded?

Reply:
Because it was failing. Someone upgraded it and it never worked. So I downgraded back to the last working version by looking at `git blame`.

Reply:
Note: this is the "Legacy Sveltkit" example, not to be confused with the new 1.0.0 example